packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -375,6 +526,15 @@ export class TokenDetectionController extends StaticIntervalPollingController<
         }
 
         if (tokensToAdd.length) {
+          this.#trackMetaMetricsEvent({
```

Comment:
Are there any tests we need to add for this change?

Reply:
I'm okay with the tests being added in #3867, so I'm resolving this conversation.